control-plane/pkg/reconciler/channel/channel.go

```diff
@@ -346,9 +346,8 @@ func (r *Reconciler) reconcileKind(ctx context.Context, channel *messagingv1beta
 		addressableStatus.Addresses = []duckv1.Addressable{httpAddress}
 	}
 
-	address := addressableStatus.Address.URL.URL()
-	proberAddressable := prober.Addressable{
-		Address: address,
+	proberAddressable := prober.ProberAddressable{
+		AddressStatus: &addressableStatus,
```

Comment:
Somehow, we missed the changes at line 322 and 340 from the original commit https://github.com/knative-extensions/eventing-kafka-broker/commit/828b81ede7af938cdffd01f6f103992469330728

Reply:
interesting, thanks for pointing this out!! I will make a PR to fix it.

Reply:
Already fixed!